docs/source/using-discovery/usingslurm.rst

```diff
@@ -3,51 +3,95 @@
+
+Overall, Slurm is a powerful and versatile tool for managing HPC resources. Its flexibility, scalability, and reliability make it an essential tool for many HPC environments.
+
+Slurm on Discovery
+--------------------
+Slurm is the software on Discovery. To be most effective, one must have a basic understanding of SLURM. This page provides knowledge needed for working on Discovery.
+
+By following the contents of this page, the user will be able to
+
+* View information about the cluster
+* Submit jobs on Discovery
+* Monitor jobs
+* View account information
+* Check the cluster state, along with that of specific nodes
+* Gain knowledge of various best practices when using SLURM
+
+This page is specific for using Slurm on Discovery. For more in-depth and general information, please refer to the official `Slurm documentation`_.
+
+
+Slurm CLI
+==============
+List of typical Slurm commands for working on Discovery.
+
+These functions are fully documented for the version of Slurm installed on in the manual pages. To get the manual pages for the Slurm functions, run ``man <SLURM_command>`` on the HPC. This command lists all the parameters and keywords that you can use with the given Slurm commands. You can also run ``<SLURM_command> --help``, which will list the help page for the Slurm function. These two give you the full possibilities of what these functions can do. The following sections are helpful/commonly used Slurm commands that are good to know for different ticket requests and common checks on the cluster.
 
 .. important::
-   Slurm commands have numerous options to help your jobs run efficiently by requesting specific resources. Options also usually have both short and verbose versions, such as
-   ``--nodes`` and ``-n`` (both mean the same thing). The examples in this documentation all use the
-   verbose version of the options for clarity, but you can use the short version if you prefer. For example, ``srun -p short -N 1 -n 1`` means the exact same thing as ``srun --partition=short --nodes=1 --ntasks=1``
-   Refer to the official Slurm documentation to get in-depth information about these commands and their options: `Slurm documentation website <https://slurm.schedmd.com/archive/slurm-17.11.6/srun.html>`_.
+   Slurm commands have numerous options to help your jobs run efficiently by requesting specific resources. Options usually have both short and verbose versions, such as ``--nodes`` and ``-n`` (both mean the same thing). The examples in this documentation all use the verbose version of the options for clarity, but you can use the short version if you prefer. For example, ``srun -p short -N 1 -n 1`` means the same thing as ``srun --partition=short --nodes=1 --ntasks=1``
+
+Controlling jobs
```

Comment:
We should probably put these sections into tables to align with the formatting throughout the rest of the page.

Reply:
This is such a small subset of all of the commands. Do you think a table is necessary? Just pointing out that there are long-hand and short-hand ways to specify inputs.

Reply:
I only think the table is necessary for consistent formatting/style. I think the information is clear as presented, it just doesn't align with the style throughout the rest of the page.

Reply:
I think I figured it out, let me know if I messed it up!